docs/4. Product Features/03. Flows/2. Flow Actions/04. Wait for the contact to respond.md

```
@@ -74,6 +74,17 @@ To accept the input if the input is present in the response criteria.
 ### 4. has only the phrase
 Accepts only if the response matches exactly what is defined in the response criteria.
 
+### 5. has multiple values
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)